remote/remote.html

```diff
@@ -0,0 +1,192 @@
+                                posArr[idx*3+2] = 0;
+                                colorArr[idx*3] = (raw_color & 255)/255;
+                                colorArr[idx*3+1] = (raw_color>>8 & 255)/255;
+                                colorArr[idx*3+2] = (raw_color>>16 & 255)/255;
+                                alphaArr[idx] = (raw_color>>24 & 255)/255;
+                                uvArray[idx*2] = cmd.vtx[i][3];
+                                uvArray[idx*2+1] = 1-cmd.vtx[i][4];
+                                idx++;
+                            }
+                            for (let i = 0; i < cmd.idx.length; ++i) {
+                                indexArray[index_idx] = cmd.idx[i] + vtx_length;
+                                index_idx++;
+                            }
+                            index_length += cmd.idx.length;
+                            vtx_length += cmd.vtx.length;
+                        }
+    
+    
+                        geometry.attributes.position.needsUpdate = true;
+                        geometry.attributes.uv.needsUpdate = true;
+                        geometry.attributes.color.needsUpdate = true;
+                        geometry.attributes.alpha.needsUpdate = true;
+                        geometry.index.needsUpdate = true;
+
+                        return all_cmds;
+                    }
+
+
+                    map.needsUpdate = true;
+                    map.minFilter = map.magFilter = THREE.NearestFilter;
+                    guniforms.tTex.value = map;
+
+                    const renderer = new THREE.WebGLRenderer();
+                    renderer.autoClear = false;
+                    renderer.setSize( window.innerWidth, window.innerHeight );
+                    document.body.appendChild( renderer.domElement );
+                    renderer.setScissorTest(true);
+
+                    var frame_counter = 0;
+                    var picture_counter = 0;
+
+
+                    /*const animate = function () {
+                        requestAnimationFrame(animate);
+                        if (frame_counter % 10 == 0) {
+                            all_cmds = updateBuffers(data[picture_counter]);
+                            let curr_base = 0;
+                            renderer.clear();
+                            for (const cmd of all_cmds) {
+                                renderer.setScissor(cmd.clp[0], window.innerHeight - cmd.clp[3], cmd.clp[2] - cmd.clp[0], cmd.clp[3] - cmd.clp[1]);
+                                geometry.setDrawRange(curr_base, cmd.cnt);
+                                renderer.render( scene, camera );
+                                curr_base += cmd.cnt;
+                            }
+                            picture_counter = (picture_counter+1) % data.length;
+                        }
+                        frame_counter++;
+                    }
+
+                    animate();*/
```

Comment:
As a general remark, do not comment code. If you  are not planning to use it immediately, simply move it into a separate function. If you do not plan to use it, drop it.